### DISCUSSION TITLE
Could not create a new project

### DISCUSSION THREAD

**Discussion by Stefan39** [symfony/symfony#44768] [Q&A] [answered]
I would like create a new project on symfony cloud as first for testing/development, fix some bugs (i.e. lighthouse, etc.) and go to production plan, but i got this message on creation:

` we are currently not able to create new projects on SymfonyCloud as we are transitioning to a new platform. If you are reading the Symfony book, you can skip the SymfonyCloud sections ("Going to production") as they ar  
  e not required to follow along  `

Anyone knows whats going wrong or how long time this will taken?

Thanks,
Stefan

**Comment by stof** [comment, marked as answer]
I suggest you to contact the SymfonyCloud support through https://symfony.com/support
SymfonyCloud is a commercial service of Symfony SAS, not a project maintained by the open-source community

**Reply by Stefan39** [reply]
Thanks @stof 

I have found another (cheaper) solution for this and use buddy.works for a CI/CD on my own cheaper Web-Host on Hetzner ;-)
